openlibrary/plugins/openlibrary/js/onboardingCard.js

```diff
@@ -47,6 +47,12 @@ export function initOnboardingCard() {
         };
 
         const goToSlide = function (slide) {
+            if (slide === maxSlide-1) {
```

Comment:
`===` is a strict comparison operate so both operands should be of same type. The value returned by e.target.data is string.